src/urdf.jl

```diff
@@ -134,9 +135,13 @@ struct URDFVisuals <: AbstractGeometrySource
     package_path::Vector{String}
     file_path::String
     tag::String
+    link_colors::Dict{String, RGBA{Float32}}
+end
+
+function URDFVisuals(xdoc::XMLDocument; package_path=ros_package_path(), file_path="", tag="visual", link_colors=Dict{String, RGBA{Float32}}())
```

Comment:
Can we relax the color type to just `<: Colorant` ? The various color types all convert nicely from one to another, so it should be fine for a user to pass in any kind of color type.

Reply:
Ah, this is just the default value, there's no type restriction. Added a test for this use case though.

Reply:
oh, right, my mistake